packages/crypto/src/crypto.ts

```diff
@@ -466,3 +492,142 @@ const bigIntToHex = (num: bigint, length: number): string => {
   }
   return hexString.padStart(length, "0");
 };
+
+/**
+ * Verifies a signature from a Turnkey enclave using ECDSA and SHA-256.
+ *
+ * @param {string} enclaveQuorumPublic - The public key of the enclave signer.
+ * @param {string} publicSignature - The ECDSA signature in DER format.
+ * @param {string} signedData - The data that was signed.
+ * @param {Environemnt} environment - An enum PROD or PREPROD to verify against the correct signer enclave key
+ * @returns {Promise<boolean>} - Returns true if the signature is valid, otherwise throws an error.
+ */
+
+export const verifyEnclaveSignature = async (
+  enclaveQuorumPublic: string,
+  publicSignature: string,
+  signedData: string,
+  environment?: Environment
+) => {
+  const signer_key = SIGNER_PUBLIC_KEY[environment ?? Environment.PROD];
+
+  if (enclaveQuorumPublic != signer_key) {
+    throw new Error(
+      "expected signer key does not match signer key from bundle"
+    );
+  }
```

Comment:
I don't think this is checking the bundle key? It's only matching what's arbitrarily passed in (`enclaveQuorumPublic`). Nothing guarantees it comes from the bundle.

I think a better interface would be "verifyBundle"?